SPDY/SPDYSocket.m

```diff
@@ -533,7 +533,6 @@ - (bool)connectToOrigin:(SPDYOrigin *)origin
     }
 
     [self _startConnectTimeout:timeout];
-    _flags |= kDidStartDelegate;
```

Comment:
What happens if there's a connect timeout while waiting on a PAC response? Could that cause the delegate to fail to be notified and the socket/session to enter a "zombie" state?